AcceptanceTests/DSLExtensions.swift

```diff
@@ -1,9 +1,34 @@
 import Quick
+import Nimble
+import Difference
 
 func they(_ description: String, flags: FilterFlags = [:], closure: @escaping () -> Void) {
-    it(description, flags: flags, closure: closure)
+    it("they " + description, flags: flags, closure: closure)
 }
 
 func fthey(_ description: String, flags: FilterFlags = [:], closure: @escaping () -> Void) {
     fit(description, flags: flags, closure: closure)
 }
+
+func when(_ description: String, flags: FilterFlags = [:], closure: () -> Void) {
+    context("when " + description, flags: flags, closure: closure)
+}
+
+func equalDiff<T: Equatable>(_ expectedValue: T?) -> Predicate<T> {
```

Comment:
I'd add a comment linking to the Difference readme section where this is explained, so we know where it came from. I'd also prefer the name `assertEqualWithDiff` or something else equally descriptive to improve the call site.